SF_Users_API/db/migrations/20171030225656_initial.js

```diff
@@ -0,0 +1,33 @@
+
+exports.up = function(knex, Promise) {
+  return Promise.all([
+    knex.schema.createTable('sf_users', function(table) {
+      table.increments('id').primary();
+      table.string('username');
+      table.string('email');
+      table.string('address');
+
+      table.timestamps(true, true);
+    }),
+
+    knex.schema.createTable('sf_favorites', function(table) {
+      table.increments('id').primary();
+      table.string('favorite_id');
```

Comment:
Do we know where this ID is coming from? Is it the school ID?  If so, can you change to "school_id"?